src/_test_/Home.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { MemoryRouter as Router } from 'react-router-dom';
+import store from '../redux/store';
+import HomePage from '../components/HomePage';
+
+it('It should render the homepage components correctly', () => {
+  const home = render(
+    <Router>
+      <Provider store={store}>
+        <HomePage />
+      </Provider>
+    </Router>,
+  );
+  expect(home).toMatchSnapshot();
+});
```

Comment:
- [ ] Regarding the testing requirements:

![image](https://github.com/coker2019/metrics_app/assets/77069011/401bf57f-a028-4011-94c9-f9eade1dc988)

Currently, you don't have any integration tests in your app. Please consider adding some! 🤓 🔧

Reply:
well resolved! ✔️